### PR TITLE
Add io.github.thaunknown.miru

### PR DESCRIPTION
**Pull request by kutzlect**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** [upstream issue](https://github.com/ThaUnknown/miru/issues/183)
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

@alsoGamer would also like to be a collaborator.

### CODE REVIEW

File: io.github.thaunknown.miru.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      - mv squashfs-root /app/bin/miru
+    sources:
+      - type: script
+        dest-filename: miru-run
+        commands:
+          - zypak-wrapper /app/bin/miru/miru "$@"
+      - type: file
+        path: io.github.thaunknown.miru.metainfo.xml
+      - type: file
+        path: io.github.thaunknown.miru.png
```
Comment (hfiguiere) [inline io.github.thaunknown.miru.yaml:44]: the appimage doesn't come with an icon?

File: io.github.thaunknown.miru.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      - type: file
+        dest-filename: miru.AppImage
+        url: https://github.com/ThaUnknown/miru/releases/download/v3.10.4/linux-Miru-3.10.4.AppImage
```
Comment (hfiguiere) [inline io.github.thaunknown.miru.yaml:47]: you are still building an aarch64 flatpak.

File: io.github.thaunknown.miru.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  - name: miru
+    buildsystem: simple
+    build-commands:
+      - chmod +x miru.AppImage
+      - ./miru.AppImage --appimage-extract
+      - rm miru.AppImage
+
+      - cp squashfs-root/miru.desktop io.github.thaunknown.miru.desktop
+      - desktop-file-edit --set-icon io.github.thaunknown.miru --set-key Exec --set-value
+        'miru-run %u' io.github.thaunknown.miru.desktop
```
Comment (hfiguiere) [inline io.github.thaunknown.miru.yaml:28]: you need to copy. Here just change it and then below install it from there. It's way simpler.

**Comment by hfiguiere** [comment]
it will fail

bot, build io.github.thaunknown.miru

**Comment by kutzlect** [comment]
bot, build io.github.thaunknown.miru

**Comment by kutzlect** [comment]
bot, build io.github.thaunknown.miru

**Comment by kutzlect** [comment]
bot, build io.github.thaunknown.miru

**Comment by kutzlect** [comment]
this shall not fail

bot, build io.github.thaunknown.miru

**Comment by kutzlect** [comment]
bot, build io.github.thaunknown.miru

**Comment by kutzlect** [comment]
bot, build io.github.thaunknown.miru

**Pull request closed without merging**